.github/workflows/link-checker.yaml

```diff
@@ -0,0 +1,44 @@
+name: Check Markdown links
+
+on:
+  pull_request:
+
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Get all docs files that have changed
+      id: changed-files-yaml
+      uses: tj-actions/changed-files@v45
```

Comment:
Instead of this, how about just using the [`paths` filter](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) on the `pull_request` event? This way, the Workflow won't even run if those files are not modified.. And the different `if: ` statements in the steps will not be needed anymore.

```yaml
on:
	pull_request:
		paths:
			- '**.md'
			- 'docs/**'
```

Reply:
Good point. I don't remember why I did it this way 😇 
I'll try to change it see what happens